examples/user_guide/14-Data_Pipelines.ipynb

```diff
@@ -121,15 +123,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "rolling_stream = Stream.define('rolling', rolling_window=5)\n",
-    "stream = rolling_stream()"
+    "import panel as pn\n",
+    "\n",
+    "slider = pn.widgets.IntSlider(name='rolling_window', start=1, end=100, value=50)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we can define a function that both loads the symbol and applies the ``rolling`` operation passing our ``rolling_window`` parameter to the operation:"
+    "Using the ``.apply`` method we can now apply the ``rolling`` operation to the DynamicMap and set the ``value`` parameter of the slider to supply the rolling_window value. Note that as long as the applied function is a HoloViews ``Operation`` we could also apply it directly but for pure functions we have to use ``.apply``."
```

Comment:
> Note that as long as the applied function is a HoloViews ``Operation`` we could also apply it directly but for pure functions we have to use ``.apply``.

I find the phrasing here a little odd: I figured out what you mean but I had to re-read it a few times. I think the subject of the sentence should be the apply method and what it supports (HoloViews operations..but also regular functions). I don't have a concrete suggestion right now so I am happy to leave this as is if we can't think of some better phrasing...